prow/cluster/tls-ing_ingress.yaml

```diff
@@ -15,14 +15,15 @@
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
+  name: prow-ing
+  namespace: default
   annotations:
     cert-manager.io/cluster-issuer: letsencrypt-prod
     kubernetes.io/tls-acme: "true"
     acme.cert-manager.io/http01-edit-in-place: "true"
-  name: prow-ing
-  namespace: default
+    kubernetes.io/ingress.class: "gce"
```

Comment:
**Unrelated:** GCP still doesn't support `ingressClassName`... and OpenShift 4.12 onwards will spam you of alerts saying that you can have an ingress without `ingressClassName` set... the Kubernetes ecosystem is weird

Reply:
GCP is really lacking here. It almost seems like they want you to go to Gateway API instead of fixing this.